lib/src/map/flutter_map_state.dart

```diff
@@ -42,11 +42,11 @@ class FlutterMapState extends MapGestureMixin
 
     move(options.center, zoom, source: MapEventSource.initialization);
 
-    // Funally, fit the map to restrictions
-    if (options.bounds != null) {
-      fitBounds(options.bounds!, options.boundsOptions);
-    }
     WidgetsBinding.instance.addPostFrameCallback((_) {
+      // Finally, fit the map to restrictions
+      if (options.bounds != null) {
+        fitBounds(options.bounds!, options.boundsOptions);
```

Comment:
Was this change intentional? 

https://github.com/fleaflet/flutter_map/pull/1376/files#diff-0712fe61169af14b5ce2bed28ca88305d26d4d91630a99158d64f19c6cb7283dL46